neural_nets/wrappers/wrappers.py

```diff
@@ -142,11 +142,59 @@ def backward(self, dLdy):
         dLdy *= 1.0 / (1.0 - self.p)
         return self._base_layer.backward(dLdy)
 
+class AlphaDropout(WrapperBase):
+    def __init__(self, wrapped_layer, p):
+        """
+       Alpha Dropout is a `Dropout` that aim at keeping mean and variance to
```

Comment:
It looks like this description is copied from the Tensorflow documentation. I think it's better if we have documentation be more explicit + verbose for a project like this, since the expectation is that these will be "reference" implementations for people to use when trying to understand how the different model components work.

With that in mind, this should be rewritten. I'm happy to help proofread and revise grammar as you'd like :-)